docs/developer-guide/etendo-classic/how-to-guides/how-to-create-a-hql-based-table.md

```diff
@@ -0,0 +1,171 @@
+---
+tags: 
+  - HQL
+  - Table
+  - Column
+  - Field
+---
+
+# How to create a HQL Based Table
+
+## Introduction
+
+There are three available data origin in the definition of a table in the application dictionary:
+
+  * Table. A table actually defined in a DBMS.
+  * Datasource. A manual datasource will be used for the data origin of this table.
+  * HQL Table.
+
+The _HQL Table_ data origin allows to define a table without the need of a physical table in the database and without having to define a manual datasource.
+
+HQL Based Tables must define a HQL select query, that will be used to obtain the proper records when a request to its datasource is done.
+
+## Defining a HQL Based Table in the Application Dictionary
+
+### Table Definition
+
+When a HQL based table is defined in the application dictionary, there is no need to fill in the _DB Table Name_ and _Java Class Name_ fields. There are two new fields that must be set for these tables:
+
+  * HQL Query. The HQL query that will be used to fetch the records for this table.
+  * Entity Alias. The alias of one of the tables used in the HQL Query. It will be used to add automatically the mandatory client and organization filters.
```

Comment:
```suggestion
  * Entity Alias: The alias of one of the tables used in the HQL Query. It will be used to add automatically the mandatory client and organization filters.
```